### PR TITLE
[ui][fix] fix amq-to-rest and api-client-connector features

### PR DESCRIPTION
**Pull request by mcada**
//skip-ci

the tests are blocked by 
https://github.com/syndesisio/syndesis/issues/5669
https://github.com/syndesisio/syndesis/issues/5892
https://github.com/syndesisio/syndesis/issues/5900

But they are all fixed - they just should fail on current syndesis version (1.7.4)

### CODE REVIEW

File: ui-tests/src/test/java/io/syndesis/qe/steps/customizations/extensions/ExtensionSteps.java [hunk 1]
```diff
@@ -130,10 +127,9 @@ public void uploadExtensionFromFile(String extensionName, String extensionPath)
 
         String techExtensionUrl = extensionPath + extensionName;
         Path techExtensionJar = Paths.get(techExtensionUrl).toAbsolutePath();
-        $(By.cssSelector("input[type='file']")).shouldBe(visible).uploadFile(techExtensionJar.toFile());
+        $(By.tagName("input")).should(exist).uploadFile(techExtensionJar.toFile());
```
Comment (asmigala) [inline ui-tests/src/test/java/io/syndesis/qe/steps/customizations/extensions/ExtensionSteps.java]: don't we also want it to be visible?

Comment (mcada) [reply ui-tests/src/test/java/io/syndesis/qe/steps/customizations/extensions/ExtensionSteps.java]: No, it is not visible in react ui, its a hidden element :/

Comment (asmigala) [reply ui-tests/src/test/java/io/syndesis/qe/steps/customizations/extensions/ExtensionSteps.java]: hang on, so how does the user upload the file?

Comment (mcada) [reply ui-tests/src/test/java/io/syndesis/qe/steps/customizations/extensions/ExtensionSteps.java]: Drag'n'drop area != input element. We need the input element so we can call upload method on it. Don't waste your time on WIP draft PRs ;-)

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: ui-tests/src/test/java/io/syndesis/qe/steps/customizations/extensions/ExtensionSteps.java (hunk 1 above)
Comment (asmigala) [reply ui-tests/src/test/java/io/syndesis/qe/steps/customizations/extensions/ExtensionSteps.java]: > Don't waste your time on WIP draft PRs ;-)

Well, don't request a review on them then.

Comment (mcada) [reply ui-tests/src/test/java/io/syndesis/qe/steps/customizations/extensions/ExtensionSteps.java]: Hmm I thought that the review email is not sent unless it is not a draft. My bad then.

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**